embassy-rp/src/uart/mod.rs

```diff
@@ -100,6 +100,10 @@ pub enum Error {
     Parity,
     /// Triggered when the received character didn't have a valid stop bit.
     Framing,
+    /// There was an issue when calculating the number of transferred items
+    /// in an aborted DMA transaction. This is likely an error in the
+    /// driver implementation, please open an embassy issue.
+    Calculation,
```

Comment:
for "should never happen unless there's a bug in the driver" scenarios, we should panic instead of returning an error to the caller.

Reply:
Got it! Will switch back to that.